10.9-libcxx/stable/main/finkinfo/sci/qhull6.3.1.info

```diff
@@ -1,9 +1,9 @@
 Package: qhull6.3.1
```

Comment:
Wanna turn this .info layout inside-out also, so we can kill this libversion easily soon?

Reply:
Done